cmake/dts.cmake

```diff
@@ -198,6 +198,7 @@ if(SUPPORTS_DTS)
   --bindings-dirs ${DTS_ROOT_BINDINGS}
   --conf-out ${DEVICETREE_CONF}
   --header-out ${DEVICETREE_UNFIXED_H}
+  --dts-out ${PROJECT_BINARY_DIR}/merged.dts # As a debugging aid
```

Comment:
This is still generating `merged.dts` rather than `zephyr.dts`.

Reply:
Oops... search fail. Fixed.

Had missed a spot in the docs too.